### ISSUE TITLE
Milestone 1 Feedback

### ISSUE THREAD

**Issue by samglover03** [bgallll/cisc181finalproject#1]
I can't seem to find a deployed website link or any deployments on your final project Github. Make sure you deploy your website, so I can see your website's progress. If you are having any issues with this, you can let me know.

**Comment by bgallll** [comment]
I don't see the Deploy dev build on main push option. is just shows this:
<img width="833" alt="Screenshot 2024-05-04 at 7 25 06 PM" src="https://github.com/bgallll/final-project-baxter-gallagher/assets/148893976/aa8706e9-f01f-4fd0-b850-8eb84d16e4ed">

**Comment by samglover03** [comment]
I see a deployment now for your site, but now there seems to be an issue with your deployment. I'll attach a screenshot of what pops up when I click the link. Did you happen to touch/edit any of the given source files?

<img width="713" alt="Screenshot 2024-05-05 at 2 18 57 PM" src="https://github.com/bgallll/final-project-baxter-gallagher/assets/93454726/ec72ff19-8e20-4f88-9cd6-6e01eae04bd5">

**Comment by bgallll** [comment]
No, I don't believe I did, and when I run "npm test start" it all works perfectly.

**Comment by samglover03** [comment]
Have you ran "npm run start" because that's the command that you should be running

**Comment by bgallll** [comment]
sorry, I meant "npm run start". this command works for me

**Comment by samglover03** [comment]
Ok so "npm run start" is used to run your site locally on your computer, but your deployed site which is at this link: https://bgallll.github.io/final-project-baxter-gallagher/  seems to not be working. The deployed site is what everyone can use to see your website, so right now I can't see your website because it is only working locally on your computer. I think one of your files must have gotten messed up. Let me know if you have trouble figuring your deployment out, and I can try to figure it out.

**Comment by bgallll** [comment]
Hi, I've tried a lot but I'm definitely having trouble deploying it.

**Comment by samglover03** [comment]
Ok I'll take a look at it

**Comment by samglover03** [comment]
It looks like you didn't clone the final project assignment from Github. 
Click this link https://classroom.github.com/a/PpxBwu6A to clone the right repository and use it to finish doing your final project. You're gonna have to take whatever component code you had in this repository and put it in the right one.

**Comment by bgallll** [comment]
Okay thank you
